src/frontend/web_application/src/scenes/ApplicationSettings/components/InterfaceSettings/presenter.jsx

```diff
@@ -28,14 +27,14 @@ class InterfaceSettings extends PureComponent {
     const { i18n } = this.props;
     this.i18n = {
       fr_FR: i18n._('settings.interface.language.options.fr', null, { defaults: 'French' }),
```

Comment:
Shouldn't the language name be the only untranslated string ?

Reply:
:+1:  maybe something like «Français (French)»
I don't know if there is a RFC/bestpractice/lib… that can help